nss-day-cohort-47/tabloidfullstack-eureka
# Js create tag

# Description
Allow users to create a tag.
# Testing Instructions
Start application, select tag management, create tag.  Enter tag name, select submit.  new tag should populate on the list. 
# Checklist:
- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] My changes generate no new warnings or errors
- [x ] I have added test instructions that prove my fix is effective or that my feature works
